Documentatie.md

```diff
@@ -101,3 +102,11 @@ _AANBEVELING_  Geef de bron `attribution` op in een TileJSON bestand, zodat deze
 Verder raden wij aan ook te kijken naar de [[API-Designrules]] voor de Nederlandse overheid. Deze bevatten principes voor het aanbieden van APIs door Nederlandse overheidsorganisaties.
 
 _AANBEVELING_ Volg de aanbevelingen over documentatie uit de [[API-Designrules]] voor de Nederlandse overheid: https://docs.geostandaarden.nl/api/API-Designrules/#documentation
+
+### Aanbeveling: tile bestandsgrootte
```

Comment:
Denk in deze paragraaf na over consistent gebruik van tegel versus tile. Gezien de rest van de tekst als zelfstandig naamwoord wel _tegel_ wordt gebruikt.

Reply:
Voorstel voor regel 107/108: 
```
De bestandsgrootte van rastertiles wordt beprerkt door het aantal pixels (breedte en hoogte), de kleurdiepte en de variatie in pixelwaarden. De bestandsgrootte van vectortiles wordt bepaald door meerdere factoren:
- het aantal objecten dat in een vectortile zitten
- het aantal coördinaten dat de contouren van die objecten beschrijft
- het aantal atributen van het object
- de attribuutwaarden van het object

Hierom is het wenselijk om de bestandsgrootte van vectortiles in de gaten te beperken, bijvoorbeeld door objecten te selecteren, geometrisch te generaliseren, attributen te verwijderen of attribuutwaarden te vereenvoudigen.
```

Reply:
@emacgillavry @skinkie @jlschaap ik wilde dit gaan overnemen in de praktijkrichtlijn, maar realiseer me nu ook dat dit dan misschien beter past in de best practices zoals ik ze nu hier doorlopen heb: https://github.com/RoelvandenBerg/vector-tiling-best-practices/blob/roelvandenberg-aanvullingen/BestPractices.md#aanbeveling-optimalisatie-tegel-set

wat denken jullie?

Reply:
Als jullie dat vinden pas ik dat volgende week nog aan en merge ik jouw opmerkingen daar.